inc/saiswitch.h

```diff
@@ -3468,6 +3468,57 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_NEXT_HOP_USER_META_DATA_RANGE,
 
+    /**
+     * @brief Linkup polling time range (in secs)
+     *
+     * @type sai_u16_range_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_FAST_LINKUP_POLLING_TIME_RANGE,
+
+    /**
+     * @brief Time (in sec) during which the fast link-up is attempted.
+     *
+     * If this timer expires before the link is UP the regular link-up will be performed
+     * Supported range can be obtained using SAI_SWITCH_ATTR_FAST_LINKUP_POLLING_TIME_RANGE
+     *
+     * @type sai_uint16_t
+     * @flags CREATE_AND_SET
+     * @default 60
+     */
+    SAI_SWITCH_ATTR_FAST_LINKUP_POLLING_TIME,
+
+    /**
+     * @brief Linkup guard time range (in secs)
+     *
+     * @type sai_u16_range_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_FAST_LINKUP_GUARD_TIME_RANGE,
+
+    /**
+     * @brief Time (in secs) during which the link must be UP with the BER below the level configured with SAI_SWITCH_ATTR_FAST_LINKUP_GUARD_TIME_BER to keep the fast link-up configuration.
+     *
+     * If either link failures happens within this time or high BER is measured at the end of this period of time the link should undergo regular link up process
+     * Supported range can be obtained using SAI_SWITCH_ATTR_FAST_LINKUP_GUARD_TIME_RANGE
+     *
+     * @type sai_uint8_t
+     * @flags CREATE_AND_SET
+     * @default 10
+     */
+    SAI_SWITCH_ATTR_FAST_LINKUP_GUARD_TIME,
```

Comment:
@iamjustadd @eddyk-nvidia since this is a timeout value, can this be renamed to `SAI_SWITCH_ATTR_FAST_LINKUP_GUARD_TIMEOUT` ?